src/modules/displaytasks.js

```diff
@@ -0,0 +1,73 @@
+import listTask from './listTask.js';
+/* eslint-disable */
+const todo = new listTask();
+/* eslint-enable */
+
+const listSection = document.querySelector('.tasks');
+
+//  show tasks
+
+const displayTasks = () => {
+  listSection.replaceChildren();
+  if (todo.allTasks.length > 0) {
+    listSection.style.display = 'block';
+    const listContainer = document.createElement('ul');
+    listContainer.className = 'item';
+    listSection.appendChild(listContainer);
+    todo.allTasks.map((a) => {
+      const list = document.createElement('li');
+      list.className = 'itme-desc';
+
+      const descrptContainer = document.createElement('div');
+      descrptContainer.className = 'task-div';
+      const checkbox = document.createElement('input');
+      checkbox.type = 'checkbox';
+      checkbox.id = 'checkbox';
+      if (a.completed === true) {
+        checkbox.checked = 'checked';
+      }
+
+      checkbox.onclick = (e) => {
+        todo.completedTask(e.target.checked, a.index);
+      };
+      descrptContainer.appendChild(checkbox);
+
+      const descrpt = document.createElement('p');
+      descrpt.id = 'taskLabel';
+      descrpt.textContent = a.description;
+      descrptContainer.appendChild(descrpt);
+      list.appendChild(descrptContainer);
+
+      descrpt.addEventListener('keyup', (e) => {
+        if (e.key === 'Enter') {
+          todo.editTask(descrpt, a.index);
+        }
```

Comment:
I think you can convert the following codes to a single line if statement and remove the brackets. This would make your code shorter.

Reply:
Done accordingly.